src/Maker/MakeEntity.php

```diff
@@ -820,9 +819,11 @@ private function askRelationType(ConsoleStyle $io, string $entityClass, string $
         return $io->askQuestion($question);
     }
 
-    private function verifyEntityName(string $entityName): bool
+    private function verifyEntityName(string $entityName): array
     {
-        return preg_match('/^[a-zA-Z\\\\]+$/', $entityName);
+        preg_match('/[^\x00-\x7F]/u', $entityName, $matches);
```

Comment:
```suggestion
        preg_match('/([^\x00-\x7F]+)/u', $entityName, $matches);
```

Reply:
We just need to know if the entity name contains any non-ascii's chars. Otherwise, we'd ask the questions multiple times which isnt very DX friendly...